spec/minfraud/transaction_spec.rb

```diff
@@ -14,7 +14,7 @@
     end
 
     it 'raises an exception if required attributes are not set' do
-      expect { Minfraud::Transaction.new { |c| true } }.to raise_exception(Minfraud::TransactionError, /required/)
+      expect { Minfraud::Transaction.new { |c| true } }.to raise_exception(Minfraud::TransactionAttributesMissingError, /required/)
     end
 
     it 'raises an exception if string attributes are invalid' do
```

Comment:
Test name+test can be changed to -> 

`raises TransactionAttributeValidationError  when string attributes are invalid and strong_validation flag is true`

Reply:
Same can be said for related tests